hazelcast-jet-spring/src/main/java/com/hazelcast/jet/spring/JetClientBeanDefinitionParser.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.spring;
+
+import com.hazelcast.jet.Jet;
+import com.hazelcast.spring.HazelcastClientBeanDefinitionParser;
+import org.springframework.beans.factory.support.AbstractBeanDefinition;
+import org.springframework.beans.factory.xml.ParserContext;
+import org.w3c.dom.Element;
+
+import static org.springframework.beans.factory.support.BeanDefinitionBuilder.rootBeanDefinition;
+
+/**
+ * BeanDefinitionParser for Hazelcast Jet Client Configuration.
+ * <p>
+ * <b>Sample Spring XML for Hazelcast Jet Client:</b>
+ * <pre>{@code
+ *   <jet:client id="client">
+ *      <jet:group name="jet" password="jet-pass"/>
```

Comment:
group password is now removed

Reply:
not yet for spring https://github.com/hazelcast/hazelcast/blob/master/hazelcast-spring/src/main/resources/hazelcast-spring-3.10.xsd#L2398.

Reply:
created a PR https://github.com/hazelcast/hazelcast/pull/12463